tools/imex-opt/imex-opt.cpp

```diff
@@ -38,3 +39,22 @@ int main(int argc, char **argv) {
   return ::mlir::asMainReturnCode(
       ::mlir::MlirOptMain(argc, argv, "Imex optimizer driver\n", registry));
 }
+
```

Comment:
Would be better if registration could be done through "::imex::registerAllPasses()" automatically by using files like Passes.td Passes.h but could be done in a follow up PR.